jest.config.js

```diff
@@ -91,6 +91,8 @@ module.exports = {
     'src/components/screens/monitor/delegates/delegateRow.js',
     'src/components/screens/monitor/delegates/delegates.js',
     'src/components/screens/monitor/network/map.js',
+    'src/components/screens/monitor/delegates/latestVotes',
+    'src/components/screens/monitor/delegates/delegatesTable/index.js',
```

Comment:
I'd expect some comment (here or in PR description) explaining why the test coverage is ignored. E.g. there is a follow-up ticket to write the tests, or some other reason.

Reply:
There's not much to it. These tests are extremely dull. I'll cover them later on when I have more time. plus, there's an epic for adding e2e tests soon.